### ISSUE TITLE
OpenNLP 2.0.0

### ISSUE THREAD

**Issue by sergey-tihon** [sergey-tihon/OpenNLP.NET#17]
We currently cannot upgrade to `OpenNLP 2.0.0` because is it compiled with `JDK 11` but `IKVM.NET` currently support only `JDK 8`

```
Build-Jdk-Spec: 11
Bundle-RequiredExecutionEnvironment: JavaSE-1.8
```

Issues for tracking progress 
- https://github.com/ikvm-revived/ikvm/issues/119
- https://github.com/ikvm-revived/ikvm/issues/120
- https://github.com/ikvm-revived/ikvm/issues/127

**Comment by GeorgeS2019** [comment]
@wasabii 
When do you think YOU would accelerate to support JDK 11 :-)

**Comment by sergey-tihon** [comment]
<img width="457" alt="image" src="https://user-images.githubusercontent.com/1197905/186652919-f497983a-7cea-46e1-a0ad-424409dc7ef1.png">
<img width="450" alt="image" src="https://user-images.githubusercontent.com/1197905/186653227-a7618df8-1661-4b82-95e7-be09791472f3.png">

@GeorgeS2019 This path will not be easy

**Comment by GeorgeS2019** [comment]
@sergey-tihon  you need to join his DISCORD to see that HE has a working Visual Studio 2022 compiling Java! 
His next step is to develop ikvmcsproj template and java code intellisense.
He just need a bit more "support" to make more magic happens :-)

**Comment by wasabii** [comment]
@sergey-tihon Neat.

It would be useful to have some other people besides myself understanding how the NuGet publishing of projects that contain MavenReferences works. It's a complicated thing to reason through, and could have consequences I haven't considered.

**Comment by NightOwl888** [comment]
True. I have been meaning to put together a test to build a NuGet package that depends on both a MavenReference and a PackageReferece that depends on MavenReference to see if its consumers correctly build both MavenReferences. I am sure transitive dependencies work on their own, but have doubts that multiple levels of transient dependencies will properly merge in all consumers regardless of depth.

**Comment by sergey-tihon** [comment]
> It would be useful to have some other people besides myself understanding how the NuGet publishing of projects that contain MavenReferences works.

I am honest myself with myself, I do not know how it work😁. I've built all packages by hand from F# script. Project with `MavenReferences` was added to test to check that auto-compiled dlls are correct and existing tests pass with them.

@wasabii  Do you think that there is a need to rebuild & republish packages to NuGet? Or consumers should use `MavenReferences` directly in the their projects?

**Comment by wasabii** [comment]
Everybody should use MavenReference if they depends on a library in Maven
